build/webpack.common.js

```diff
@@ -128,38 +131,54 @@ const withImageModuleLoader = returnModuleRuleWithConfig(
   ]
 );
 
-// exported helper function - returns common configuration, merged via object spread. Provided options take precedence
-const returnBasicConfigMergedWith = (customConfigurationForBuildMode = {}) => ({
-  entry: `${BOOTSTRAP_DIR}/index.js`,
-  output: {
-    path: BUILD_DIR,
-    publicPath: '',
-    filename: '[name].bundle.js',
-  },
-  module: {
-    // given the dynamic nature of rules and configuration, none a re provided by default. Use the exported `moduleLoaders` to return loaders with some default configs
-    rules: [],
-  },
-  // given different build modes will require different plugins/plugin configuration, none are provided by default. Use the exported `plugins` for commonly used plugins with default configuration
-  plugins: [],
-  resolve: {
-    alias: webpackAliases,
+const withTypescriptModuleLoader = returnModuleRuleWithConfig(
+  {
+    test: /\.tsx?$/,
+    exclude: /node_modules/,
   },
-  ...customConfigurationForBuildMode,
-});
+  ['ts-loader']
+);
```

Comment:
Do `.ts` files need to be passed through this as well? Or is it something specific to`.tsx`?

Reply:
I believe `.tsx?` regex catches both `.ts` and `.tsx`

Reply:
Apologies, I missed the `?`